### PR TITLE
EOS-18920: Patch-1: create separate validation modules and remove from configure_setup api

### PR DESCRIPTION
**Pull request by divya-vijay**
Signed-off-by: Divya Vijayakumar <divya.vijaykumar@seagate.com>

### CODE REVIEW

**Review by ypise — APPROVED** [review]

File: api/python/provisioner/commands/validator/__init__.py [hunk 1]
```diff
@@ -14,25 +14,31 @@
 # For any questions about this software or licensing,
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 #
+
 from .validator import (FileValidator,
                         DirValidator,
                         FileSchemeValidator,
                         YumRepoDataValidator,
                         HashSumValidator,
-                        YumRepoDataValidator,
```
Comment (andkononykhin2) [inline api/python/provisioner/commands/validator/__init__.py:22]: why it was removed?

**Review by andkononykhin2 — CHANGES_REQUESTED** [review]
@divya-vijay Thanks. Could you please just address few comments?

File: api/python/provisioner/commands/validator/network.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+# Module with all network-related validations
+
+
+import logging
+
+from ...inputs import (
```
[the remark this answers is not in the document]
Comment (divya-vijay) [reply api/python/provisioner/commands/validator/network.py]: yes, thought so :) But in existing `commands/validators/validator.py`, it was this way. so, kept it as such to maintain uniformity. Addressed now.

File: api/python/provisioner/commands/validator/__init__.py (hunk 1 above)
Comment (divya-vijay) [reply api/python/provisioner/commands/validator/__init__.py:22]: Duplicated twice, so removed once.

Comment (andkononykhin2) [reply api/python/provisioner/commands/validator/__init__.py:22]: Oh, right. Thanks

File: api/python/provisioner/commands/validator/network.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (andkononykhin2) [reply api/python/provisioner/commands/validator/network.py]: I see. We missed that before. Thank you for the update.

**Review by andkononykhin2 — APPROVED** [review]

**Pull request merged**